build.bat

```diff
@@ -1,5 +1,7 @@
 @ECHO OFF
 
+echo [   INFO] Build starting at %TIME% %DATE%
```

Comment:
Consistency is good. I didn't add it to `build` because I have no way to test it. Will the same syntax work on a Mac? If not, if one of you will provide the correct syntax, I'm happy to add it.